TomoPool.sol

```diff
@@ -633,7 +633,9 @@ contract CandidateContract {
         return indexes[_blockNumber] == 0;
     }
 
-    function () external payable {}
+    function () external payable {
+        stake();
```

Comment:
this should not because fallback here is for receiving withdrawals from TomoValidator contract.

Reply:
ah, ok